src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'NRQL Alert Conditions, with no NRQL required'
+summary: 'Now create alert conditions directly from the Query Builder'
+releaseDate: '2021-05-28'
+getStartedLink: 'https://one.newrelic.com/launcher/nr1-core.home?packages=nrai@0.2.417-c55ed87e&platform[timeRange][duration]=259200000&platform[$isFallbackTimeRange]=true&pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&overlay=eyJuZXJkbGV0SWQiOiJ3YW5kYS1kYXRhLWV4cGxvcmF0aW9uLmRhdGEtZXhwbG9yZXIiLCJpbml0aWFsQWN0aXZlSW50ZXJmYWNlIjoiY2hhcnRCdWlsZGVyIiwiaW5pdGlhbEFjY291bnRJZCI6MTYwNjg2MiwiaW5pdGlhbExpbWl0IjoxMCwiaW5pdGlhbFRpbWVXaW5kb3ciOnsic3RhcnQiOm51bGwsImVuZCI6Im5vdyIsIm9mZnNldCI6MTgwMDAwMH0sImluaXRpYWxUaW1lV2luZG93T3ZlcnJpZGUiOm51bGwsImluaXRpYWxGaWx0ZXJzIjoiIiwiaW5pdGlhbFBsb3RBZ2dyZWdhdGUiOnsicGxvdCI6bnVsbH0sImluaXRpYWxGYWNldCI6bnVsbCwiaW5pdGlhbENvbXBhcmVXaXRoIjpudWxsfQ=='
+---
+With the new ability to create an alert condition directly within the query builder, configuring alerts to cover all of your services and entities continues to get easier. If you are not comfortable with the New Relic Query Language (NRQL), you can create basic queries using the query builder tool, and then create an alert condition from the chart that you create. 
+
+The autocomplete functionality of the query builder lets you explore your data and create alerts without the need to know anything about NRQL. Alternatively, you can use the Data Explorer to browse the metrics and event data in your account, click your way to a chart, and create and create an alert condition from there as well. 
```

Comment:
I'd organize the sentences/paragraphs a little differently (see below).I'd also add links to the docs for the query builder and data explorer.

With the new ability to create an alert condition directly within the query builder, configuring alerts to cover all of your services and entities continues to get easier. 
If you are not comfortable with the New Relic Query Language (NRQL), you can create basic queries using the query builder tool, and then create an alert condition from the chart that you create. The autocomplete functionality of the query builder lets you explore your data and create alerts without the need to know anything about NRQL. 
Alternatively, you can use the Data Explorer to browse the metrics and event data in your account, click your way to a chart, and create and create an alert condition from there as well.

Reply:
@urbiz-nr , good suggestion! I'll do this!